test/functional/wallet_send.py

```diff
@@ -168,49 +169,91 @@ def run_test(self):
         self.nodes[1].createwallet(wallet_name="w1")
         w1 = self.nodes[1].get_wallet_rpc("w1")
         # w2 contains the private keys for w3
-        self.nodes[1].createwallet(wallet_name="w2")
+        self.nodes[1].createwallet(wallet_name="w2", blank=True)
         w2 = self.nodes[1].get_wallet_rpc("w2")
+        xpriv = "tprv8ZgxMBicQKsPfHCsTwkiM1KT56RXbGGTqvc2hgqzycpwbHqqpcajQeMRZoBD35kW4RtyCemu6j34Ku5DEspmgjKdt2qe4SvRch5Kk8B8A2v"
```

Comment:
In commit "Fix wallet_send.py wallet setup to work with descriptors" (1fa9bbdb4076944272d5246d1140d29ba3ecb3a3)

This seems like the same code earlier copied from the feature_notifications test. You can see earlier comment, but I think it would be better to have a framework or util function that can easily create wallets with the same seed. (It would also seem better not to hard code that seed, but the hardcoding is less a problem than the complexity and boilerplate these changes add to the tests)

Reply:
Done